src/Forms/Main.cs

```diff
@@ -1088,10 +1088,16 @@ private void Main_Load(object sender, EventArgs e)
                 if (screen.Bounds.Height < Height)
                     Height = screen.Bounds.Height;
 
-                if (screen.Bounds.X + screen.Bounds.Width - 200 < Left)
-                    Left = screen.Bounds.X + screen.Bounds.Width - Width;
-                if (screen.Bounds.Y + screen.Bounds.Height - 100 < Top)
-                    Top = screen.Bounds.Y + screen.Bounds.Height - Height;
+                // Fix main window coordinate (Multi-Monitor issue)
+                if (screen.Bounds.Right < Left)

```

Comment:
What was the problem exactly?
I'll test it next week (I can test multi screen setup then)